katportalclient/client.py

```diff
@@ -1263,23 +1302,18 @@ def sensor_detail(self, sensor_name):
             # check for exact match, before giving up
             for result in results:
                 if result['name'] == sensor_name:
-                    raise tornado.gen.Return(result)
+                    result = dict_to_sensor_info(result)
+                    raise tornado.gen.Return(result._asdict())
```

Comment:
Good to have a function that pulls out the interesting keys, but agree it is a little unnecessary to switch types.

The `SensorInfo` named tuple is only used internally, so it doesn't have much value.  Docstring of this method already provides details of the various fields.  If we were changing the return type to `SensorInfo` it would make more sense to add the new class, but that would break the public API.

Reply:
Good question. 
There was no definition for the structure or what constitutes the sensor information we receive from katstore the and when the changes are to katstore there are no real ways to communicate it to the client. Perhaps the `SensorInfo` is not the correct way to address it. I will revert this and bring back the old ways.